tools/project_manager/word_freq_table.inc

```diff
@@ -51,6 +55,11 @@ function get_cutoff_script($cutoffOptions,$instances) {
         }
         if(document.getElementById("freqCutoffValue")) document.getElementById("freqCutoffValue").value=newCutoff;
         document.getElementById("current_cutoff").innerHTML=newCutoff;
+
+        // set new value for forms so the value is persisted through actions
+        $('form input[name=freqCutoff').each(function (index, value) {
```

Comment:
Should this have a `]` in it? i.e. `$('form input[name=freqCutoff]')`?

Reply:
Yes, I'm actually very surprised it works without it.